easybuild/easyconfigs/n/netcdf4-python/netcdf4-python-1.2.6-intel-2016a-Python-3.5.1.eb

```diff
@@ -0,0 +1,25 @@
+easyblock = 'PythonPackage'
```

Comment:
Not having a dedicated numpy dep is fine, obviously.

The easyblock makes sure that the required deps are there (which is sort of relevant if they are there, but fine), but it also runs the tests and has a specific sanity check, so it makes a lot of sense to actually use it.